storybook/cypress/e2e/scroll-container.cy.ts

```diff
@@ -0,0 +1,34 @@
+describe("Scroll container", () => {
```

Comment:
"Scroll Container"

Reply:
En ik zou nog wel heel graag tests voor de dynamic content willen zien: We moeten de MutationObserver en ResizeObserver routines ook dekken.